aurci/general.py

```diff
@@ -49,9 +49,19 @@ def build_metainfo_dict():
             pkg_list = d.get('release', {'packages': [repo]}).get('packages', [repo])
             for pkg in pkg_list:
                 siblings = len(pkg_list)-1
+                # TODO: Replace hard-coded melodic
                 pkgname = 'ros-melodic-{}'.format(re.sub('_', '-', pkg))
+                pkg_url = '/'.join([re.sub(r'\.git', '', src),
+                                    'archive/release/melodic',
+                                    pkg,
+                                    '${pkgver}.tar.gz'])
+                pkg_dl = '/'.join([re.sub(r'\.git', '', src),
```

Comment:
Not sure if the escape backslash in the rawstring is correct, but I could be wrong.

Reply:
This should be fine, as escaping the `.` in the regular expression should be necessary.